src/fidesops/task/graph_task.py

```diff
@@ -663,7 +675,7 @@ def collect_tasks_fn(
                 data[tn.address] = GraphTask(tn, resources)
 
         env: Dict[CollectionAddress, Any] = {}
-        traversal.traverse(env, collect_tasks_fn)
+        end_nodes = traversal.traverse(env, collect_tasks_fn)
```

Comment:
I save the end_nodes here so I can pass them in to build a graph comparison between this run and the previous run where applicable.

We previously didn't return the end nodes when building the graph because we don't need them here when feeding them to dask, unlike the access request.  We just pass all of the nodes as end nodes. The erasure graph isn't even a graph really, all collections could be run in parallel in theory. We've already collected the data we need to collect in the access step.